scripts/smoothwall/README.md

```diff
@@ -0,0 +1,38 @@
+# Yona Smoothwall Log Parser
+
+This program will watch the output of the danguardian access logs for the json data with the categorization.  It will cross reference this against a list of categories to watch (via API request) and will forward the payloads onto the Analysis server.
```

Comment:
Spelling error: Should be DansGuardian instead of danguardian

Reply:
Fixed